concent_api/core/views.py

```diff
@@ -93,6 +99,17 @@ def receive(request, _message):
         set_message_as_delivered(last_undelivered_message_status)
         return handle_receive_force_get_task_result_upload_for_provider(request, decoded_message_data)
 
+    if isinstance(decoded_message_data, message.concents.ForceSubtaskResults):
+        set_message_as_delivered(last_undelivered_message_status)
+        if current_time < decoded_message_data.timestamp + settings.CONCENT_MESSAGING_TIME:
+            return handle_receive_force_subtasks_results(decoded_message_data)
+        return None
+
+    if isinstance(decoded_message_data, message.concents.SubtaskResultsAccepted) or isinstance(decoded_message_data, message.concents.SubtaskResultsRejected):
```

Comment:
You can pass a tuple of classes to `isinstance()` like this:
```
isinstance(decoded_message_data, (message.concents.SubtaskResultsAccepted, SubtaskResultsRejected))
```

Reply:
:+1: